Sources/StytchCore/SharedModels/AuthenticationFactor.swift

```diff
@@ -1,11 +1,11 @@
 import Foundation
-
+@preconcurrency import SwiftyJSON
 /**
  A type which describes an factor used to authenticate a session.
  E.g. An email which was used to log in, or a phone which was used
  via SMS as an OTP second factor.
  */
-public struct AuthenticationFactor: Hashable, Sendable {
```

Comment:
I could not determine any reason why this struct conformed to `Hashable` so I removed it so it was giving me compiler warnings.

Reply:
Ok I figured out why it was conforming to hashable and it was because something was using it in the consumer workbench.

Reply:
So I just made an extension that generated the desired string.